docs/docs.less

```diff
@@ -9,6 +9,10 @@
 @demo-fill-color: fade(@orange-500, 25%);
 @demo-padding-color: @magenta-100;
 
+[v-cloak] {
```

Comment:
Do you need to leave a comment for the future explaining why this needs to come earlier?

Reply:
Not really. It's more personal preference.